lib/lita/handlers/reviewme.rb

```diff
@@ -96,6 +96,7 @@ def comment_on_github(response, room: get_room(response))
         rescue Octokit::Error
           response.reply("Unable to check who issued the pull request. Sorry if you end up being assigned your own PR!")
         end
+        return response.reply('Sorry, no reviewers found') unless reviewer
```

Comment:
I tried this out to answer my own question and it seems to go ahead and self-assign in the case that you are the only reviewer. So in that case, why return at this point in the method?